magic-square/src/magic_square/puzzle.clj

```diff
@@ -1,8 +1,26 @@
 (ns magic-square.puzzle)
 
-(def values [1.0 1.5 2.0 2.5 3.0 3.5 4.0 4.5 5.0])
-
 (defn magic-square [values]
   [[1.0 1.5 2.0]
    [2.5 3.0 3.5]
-   [4.0 4.5 5.0]])
+   [4.0 4.5 5.0]]
```

Comment:
In clojure forms, only the value from  the last form is returned.
Therefore you should remove the array [1.0 1.5 2.0 ...], it is only shown as an example